README.md

```diff
@@ -107,6 +109,14 @@ export BEE_DEBUG_API_URL=http://localhost:1635
 npm run start
 ```
 
+#### 5. Reupload pinned content (Live Bee API)
```

Comment:
What do you mean by "Live Bee API"?

Reply:
Port 1633 instead of 1635 on Bee API URL. That should've been removed when I took out the BEE_DEBUG_API_URL of the mandatory env vars required